packages/ui/src/translations.ts

```diff
@@ -1,14 +1,27 @@
 export const BASE_TRANSLATIONS = {
 	close: "Close",
 	"show-all": "Show more search results",
+	"aria-show-all": "Show all search results in the group {{group}}",
 	"all-results-shown": "All results shown",
 	"load-more": "Load more",
 	"go-back": "Back",
 	"aria-label-close-search": "Close search",
 	"aria-label-search-input": "Search input",
 	"no-results": "No results",
+	"sr-result-count": "Got {{count}} search results for terms {{terms}}",
+	"sr-search-instructions":
+		"Search shows search results automatically as you type. Search results can be browsed with tabulator. Search searches for results in different groups and displays group's search results from best to worst. Search opens to its own window which can be closed with ESC or with Close-button at the end of the search window.",
```

Comment:
```suggestion
		"Search shows search results automatically as you type. Search results can be browsed with tabulator. Search searches for results in different groups and displays group's search results from best to worst. Search opens to its own window which can be closed with ESC or with Close-button at the start of the search window.",
```

This is outdated, Close button is located before search bar

Reply:
Removed completely. Esc is enough.